### DISCUSSION TITLE
DXF to Image (Drawing Addon) FAQ

### DISCUSSION THREAD

**Discussion by mbway** [mozman/ezdxf#550] [FAQ]
Below are a collection of answers to frequently asked questions relating to the drawing addon. See [the documentation](https://ezdxf.mozman.at/docs/addons/drawing.html) for more information.

- DXF (WCS) to image transform:  #219
- image to DXF (WCS) transform: #269
- render a section of the document: #451
- render images with an exact size: #357
- remove margins around the image: #523
- get more accurate line type rendering: #562 
- render to a specific scale: #665
- control the line width of the output #797

**Comment by chibai** [comment]
Very tricky approach to print DWG/DXF file by AutoCad~
https://github.com/mozman/ezdxf/discussions/724
